### ISSUE TITLE
Log in page says 'Social connect' but has no links or info, can we get social connect? (presumably this enables users to log in with facebook etc)

### ISSUE THREAD

**Issue by neilpie** [pie/PikaPango#28]

**Comment by neilpie** [comment]
For launch I would suggest we turn this off, but we need to discuss this aspect and profiles information on signup going forwards.

**Comment by Pikapango** [comment]
Yes it may be an idea to remove social connect at this stage, definitely a consideration for next phase in the future

**Comment by neilpie** [comment]
This needs thought with regards to how it can be implemented while also enabling a user to add the details we require. 

Possibly linked to Profile Management
